### ISSUE TITLE
Display localized metadata

### ISSUE THREAD

**Issue by linsui** [NeoApplications/Neo-Store#66]
Currently all summaries and descriptions are English only.

**Comment by k3b** [comment]
I have the same issue: 

* my android locale is de (german)
* the app gui is also german
* f-droid-catalog-data is always taken from the english catalog data not the german

**Comment by xuangeyouneihan** [comment]
Same issue too. The app gui is Chinese.
Another example, Material Files
- In F-Droid
![Screenshot_20220328_204048_org fdroid fdroid](https://user-images.githubusercontent.com/50788434/160399969-1d937c95-6ba9-410d-8df3-76ccab48aa7b.jpg)

- In Droid-ify
![Screenshot_20220328_204057_com looker droidify](https://user-images.githubusercontent.com/50788434/160400120-98ac9f83-ef70-4714-bd53-abe78ecfcf48.jpg)

**Comment by xuangeyouneihan** [comment]
Same issue too, but it appears so inconvenient for me that I can't find the app that I want to install by name!

---
For example, 质感文件/Material Files.

- In F-Droid  
![Screenshot_20220329_080852_org fdroid fdroid](https://user-images.githubusercontent.com/50788434/160528707-ebc6cc05-cf25-48ca-b5e1-6debfef6d22f.jpg)

- In Droid-ify  
![Screenshot_20220329_115654](https://user-images.githubusercontent.com/50788434/160530265-1c94048b-3f88-43af-97df-aa636c049436.jpg)

- So, when I enter "质感文件" in the search box,  
![Screenshot_20220329_115149_com looker droidify](https://user-images.githubusercontent.com/50788434/160529792-edcafdc8-fc7e-4cff-934f-7d49ffa7d01a.jpg)  
😂😂😂
---
Could you please add localized metadata display support? It is badly needed.

**Comment by xuangeyouneihan** [comment]
#166

**Comment by xuangeyouneihan** [comment]
#166

**Comment by machiav3lli** [comment]
Not planned for the foreseeable future…

**Comment by BLumia** [comment]
I tried to work on a patch to add this feature to Droid-ify so I can share the app to non-English speakers. The patch is not upstreamed to Droid-ify yet since I'm not sure if they will accept such change (due to https://github.com/Iamlooker/Droid-ify/issues/337 is currently closed as not planned).

While I found Neo-Store is also based on Foxy-droid and the related codebase seems identical, I also tried to apply the same patch to Neo-Store and it works well on my end. I'll submit the patch to this repo later.
